src/games/chess/board.tsx

```diff
@@ -126,6 +126,15 @@ export class Board extends React.Component<IBoardProps, {}> {
     for (const move of this._getMoves()) {
       result[move.to] = MOVABLE_COLOR;
     }
+    const pgnArray = this.props.G.pgn.split(' ');
```

Comment:
wow no, what you are trinng to do is extremely buggy. We have a Chess() object that you can use the Chess.js API and look the last move using the documented API instead. See here:

https://github.com/jhlywa/chess.js/blob/master/README.md
see history() specially with verbose: true

Reply:
That is much better.  Thanks!